packages/react/src/component-tests/IcSelect/IcSelect.cy.tsx

```diff
@@ -0,0 +1,2479 @@
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", "ic-menu ul li ic-typography")
+      .should(BE_VISIBLE)
+      .should("have.text", "Loading Error");
+  });
+
+  // can't mount reset button component
+  it.skip("should reset to initial value on form reset", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").shadow().find(IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.get(".ic-input").should(HAVE_VALUE, "espresso");
+  });
+
+  it.skip("menu should not be visible on initial load if setting default value and disable filter is set", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={[]}
+        searchable
+        timeout={800}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").shadow().find(IC_MENU_LI).should("not.exist");
+  });
+
+  // can't mount reset button component
+  it.skip("should reset to initial value on form reset with searchable", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={searchableCoffeeOption}
+        searchable
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.get("ic-select").shadow().find(IC_MENU_LI).should(BE_VISIBLE);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.get(".ic-input").should(HAVE_VALUE, "Lat");
+  });
+
+  //not sure
+  it("should render", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+  });
+
+  it("should render a native select when searchable is applied on mobile", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.viewport(320, 480);
+    cy.viewport("iphone-5");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+  });
+
+  it("should render when empty array is provided", () => {
+    mount(<IcSelect label="What is your favourite coffee?" options={[]} />);
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.get("ic-select").shadow().find(IC_MENU_LI).should("not.exist");
```

Comment:
this consistently fails for me. I don't think this is a valid thing to check for as there will be a li element for the no-results entry